get-started/guide-neo4j-browser/guide-neo4j-browser.adoc

```diff
@@ -55,11 +56,60 @@ You can provide your name and send us a message.
 
 == Neo4j Browser Sync
 
-Neo4j Browser Sync is a new feature that allows you to synchronize your favorites, styling and settings to a cloud storage, so you can share it across machines and browsers.
+Neo4j Browser Sync is a feature that allows you to synchronize your favorites, styling, and settings to cloud storage, so you can share it across machines and browsers.
+
+image::{img}neo4j-browser-sync.png[width=500]
+
+You just need to click the cloud icon in the left-side menu and log in with your GitHub, Twitter, or Google account and synchronize your workplace to the cloud.
+
+== Built-in Guides
+
+If you want to learn more about Neo4j, Cypher, or the Neo4j browser before diving in, click on any of the helpful links shown after startup (commands also shown below).
+These are quick guides that introduce the different concepts.
+You can find more helpful links in the left sidebar of Neo4j in the "Documents" tab (icon:book[]).
+
+[%autowidth.spread]
+|===
+|*Title* |*Description* |*Command*
+|Intro |A guided tour of Neo4j browser |`:play intro`
+|Concepts |Graph database basics |`:play concepts`
+|Cypher |Neo4j's graph query language introduction |`:play cypher`
+|The Movie Graph |A mini graph model of connections between actors and movies |`:play movie graph`
+|The Northwind Database |A classic use case of RDBMS to graph with import instructions and queries |`:play northwind graph`
+|Custom Guides |Starting with Neo4j 2.3, you can use _:play <url>_ to play a custom guide |`:play
```

Comment:
we can remove the version reference here, but we should point to: https://neo4j.com/developer/guide-create-neo4j-browser-guide/